InvoiceMaker/Models/WorkDone.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace InvoiceMaker.Models
+{
+    public class WorkDone
+    {
+        public int Id { get; private set; }
+        public DateTimeOffset StartedOn { get; private set; }
+        public DateTimeOffset? EndedOn { get; private set; }
+        public string ClientName { get; private set; }
+        public string WorkTypeName { get; private set; }
+        public int ClientId { get; private set; }
+        public int WorkTypeId { get; private set; }
+        public decimal Rate { get; private set; }
+
+        private Client Client { get; set; }
+        private WorkType WorkType { get; set; }
```

Comment:
I know all of this has changed with your EF implementation but at this point the intention was to make client and work type private fields so they weren't exposed to the outside world. Then `ClientName`, `WorkTypeName`, `ClientId`, and `WorkTypeId` would all be getter only properties returning the appropriate properties from the client and work type private fields.